portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,138 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
-
 #greeting-container {
   margin-top: 20px;
+  font-family: 'Neucha';
+  color: #FEDA15;
+
```

Comment:
please remove this blank line before the closing brace. This applies to several other places in the file.